README.md

```diff
@@ -29,6 +34,20 @@ Then:
 
 Head to the address it gives you to check if your changes have worked. More information about using mkdocs can be found [here](http://www.mkdocs.org/)
 
+## Windows SSL Certificate
```

Comment:
For clarity, similar instructions should probably also be included for Linux - just because I set up homer to allow HTTPS downloads doesn't mean every Linux machine will have this functionality. OpenSSL shouldn't need to be installed, think that'll be default on most distros, but downloading the cert file & setting the env var could need to be done if a user doesn't have this set up globally